### ISSUE TITLE
nan value in grad_norm

### ISSUE THREAD

**Issue by Alsac** [FanqingM/R1-Multimodel-Journey#14]
Parameter Offload: Total persistent parameters: 755712 in 408 params

  0%|          | 0/19788 [00:00<?, ?it/s]/pcl_data/users/wyl/ZeroTime/R1-Multimodal-Journey-main/src/open_r1/trainer/grpo_trainer_vllm.py:552: UserWarning: std(): degrees of freedom is <= 0. Correction should be strictly less than the reduction factor (input numel divided by output numel). (Triggered internally at ../aten/src/ATen/native/ReduceOps.cpp:1823.)
  std_grouped_rewards = rewards.view(-1, self.num_generations).std(dim=1)

  0%|          | 1/19788 [00:25<138:18:21, 25.16s/it]NaN or Inf found in input tensor.
NaN or Inf found in input tensor.

                                                     
{'loss': 0.0, 'grad_norm': nan, 'learning_rate': 9.999494643218111e-07, 'completion_length': 524.0, 'rewards/accuracy_reward': 0.0, 'rewards/format_reward': 0.0, 'reward': 0.0, 'reward_std': nan, 'kl': 0.0, 'epoch': 0.0}

  0%|          | 1/19788 [00:25<138:18:21, 25.16s/it]
  0%|          | 2/19788 [00:58<164:39:37, 29.96s/it]NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
